static/js/composebox_typeahead.js

```diff
@@ -321,6 +321,11 @@ exports.compose_content_begins_typeahead = function (query) {
 
     if (this.options.completions.mention && current_token[0] === '@') {
         current_token = current_token.substring(1);
+        if (current_token[0] === '*' && current_token[1] === '*') {
```

Comment:
Would it be more succinct with `current_token.startsWith('**')`?

Reply:
Yes, this is better. This also reminds about `endsWith()`. Thanks, @rht.